package.json

```diff
@@ -22,20 +22,23 @@
   ],
   "dependencies": {
     "apollo-client": "^1.4.2",
-    "babel-runtime": "^6.23.0"
+    "babel-preset-env": "^1.5.2",
```

Comment:
I still don't understand why, because the dist files are published on npm. having them in dependencies would mean they are redundantly installed in production environments.